tests/unit/test_predictor.py

```diff
@@ -466,3 +466,21 @@ def test_delete_endpoint_only():
 
     sagemaker_session.delete_endpoint.assert_called_with(ENDPOINT)
     sagemaker_session.delete_endpoint_config.assert_not_called()
+
+
+def test_delete_model():
+    endpoint_desc = {
+        'EndpointConfigName': 'my-endpoint-config'
+    }
+    endpoint_config_desc = {
```

Comment:
can we unit test an endpoint configuration with multiple models?

Reply:
Yes, will do this in next revision.